strings.go

```diff
@@ -0,0 +1,368 @@
+package main
+
+import (
+	"go/ast"
+	"go/token"
+	"log"
+	"strconv"
+	"strings"
+
+	"golang.org/x/tools/go/ast/astutil"
+)
+
+func obfuscateStrings(files []*ast.File) []*ast.File {
+
+	rmConst := func(cursor *astutil.Cursor) bool {
+		node := cursor.Node()
+
+		t, ok := node.(*ast.GenDecl)
+		if !ok {
+			return true
+		}
+
+		if t.Tok == token.CONST {
+			t.Tok = token.VAR
+		}
+
+		return true
+	}
+
+	var (
+		key        = genAesKey()
+		fset       = token.NewFileSet()
+		addedToPkg bool // we only want to inject the code and imports once
+	)
+
+	obfusStrings := func(cursor *astutil.Cursor) bool {
+		node := cursor.Node()
+
+		v, ok := node.(*ast.File)
+		if ok && !addedToPkg {
+			v.Decls = append(v.Decls, funcStmt())
+			v.Decls = append(v.Decls, keyStmt(key))
+
+			if v.Imports == nil {
+				var newDecls = []ast.Decl{
+					cryptoAesImportSpec,
+				}
+
+				for _, decl := range v.Decls {
+					newDecls = append(newDecls, decl)
+				}
+
+				v.Decls = newDecls
+			} else {
+				astutil.AddImport(fset, v, "crypto/aes")
+				astutil.AddImport(fset, v, "crypto/cipher")
+			}
+
+			addedToPkg = true
+
+			return true
+		}
+
+		if !(cursor.Name() == "Values" || cursor.Name() == "Rhs") {
+			return true // we don't want to obfuscate literals in Print Functions etc.
```

Comment:
I don't understand this. what literals do we not want to obfuscate, and why?

Reply:
We only want to obfuscate 
```go
const x = "string" // Values
var y = "string" // Values
z := "string" // Rhs
z = "another string" // Rhs
```
We dont want:
```go
import "fmt" // Path

fmt.Println("example") // Args
```

Reply:
Not obfuscating imports makes sense, thanks. You should document that.

I stlil don't get why we shouldn't obfuscate arguments. What about `T{field: "to obfuscate"}`? And what about `map[string]bool{"to obfuscate": true}`? Your test should cover more edge cases.

Reply:
Supporting obfuscating map keys `map[string]bool{"to obfuscate": true}` would take some work.
Function arguments and map struct Values now also get garbled,